pdb2sql/superpose.py

```diff
@@ -0,0 +1,242 @@
+
+import os
+import numpy as np
+from .pdb2sqlcore import pdb2sql
+from .transform import rotate
+
+
+def superpose(pdb1, pdb2, method='svd', only_backbone=True, **kwargs):
```

Comment:
I saw the PR is still in draft state. Are you refactoring the `superpose` function to be more generic? I mean the input could be `pdb2sql` and return the updated `pdb2sql`, thus the `superpose` function can be used in `StructureSimilarity.py`.

Reply:
yes I'm still working on it and I welcome input :) I also want to add the possibility to align an interface in a plane. Good point about reusing the superpose in structuresimilarity

Reply:
I can do some programming this Friday, and the other time I have planned to pair program with Hanno on a  task of generalisation team :- )